### PR TITLE
Update `XR` , `Start` and `Input` doc

### PR DESCRIPTION
**Pull request by cptbtptpbcptdtptp**
### Please check if the PR fulfills these requirements

- [ ] The commit message follows our [guidelines](https://github.com/galacean/engine/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Summary by CodeRabbit

- **New Features**
	- Introduced a guide for developing mini-programs compatible with Alipay and Taobao.
	- Added documentation for the XR Manager, detailing its role in managing XR workflows.
	- New sections on camera textures and session management introduced to enhance XR development.
	- Comprehensive input system overview for cross-platform compatibility now available.
	- Added new documentation files for Input Manager, Session Manager, and advanced tracking components.

- **Documentation**
	- Improved clarity and accuracy of existing documentation across various topics, including engine functionalities and XR capabilities.
	- Updated links and terminology for better navigation and consistency within the documentation.
	- Newly structured quick-start guide for Chinese users, featuring an overview and quick-start sections.

- **Chores**
	- Reorganized documentation structure and metadata for improved accessibility and usability.

### CODE REVIEW

File: docs/zh/xr/quickStart/debug.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+### 调试
+
+请参考[远程调试安卓设备](https://developer.chrome.com/docs/devtools/remote-debugging?hl=zh-cn)，XR 设备同理。
+
+> 在调试前确保手机开启 **`开发者选项`** ，且允许 **`USB 调试`**
+
+## 最佳实践
+
+由于 XR 调试的困难，我们建议绝大部份的工作和验证在 PC 预览与调试阶段完成，这样可以显著提升开发效率。
```
Comment (gz65555) [inline docs/zh/xr/quickStart/debug.md:57]: 部分？

File: docs/zh/xr/quickStart/develop.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    };
+  },
+  (error) => {
+    // 不支持该模式
+    console.error(error);
+  }
+);
+```
+
+## ProCode
```
Comment (gz65555) [inline docs/zh/xr/quickStart/develop.md:110]: 脚本开发更合适？

File: docs/zh/xr/system/features.md [hunk 3]
```diff
@@ -89,13 +85,19 @@ xrManager.addFeature(XRPlaneTracking, XRPlaneMode.EveryThing);
...
+  // 初始化图片追踪能力，并指定追踪图片
+  xrManager.addFeature(XRImageTracking, [refImage]);
+};
+image.src = "图片的 URL";
 ```
 
 我们可以追踪现实图片，并为他们标记坐标系：
 
-<playground src="xr-ar-planeTracking.ts"></playground>
+<playground src="xr-ar-imageTracking.ts"></playground>
```
Comment (gz65555) [inline docs/zh/xr/system/features.md:100]: 感觉 xr 放 playground 意义不大，给个项目二维码能让开发者直接体验会更加直观

File: docs/zh/xr/quickStart/develop.md (hunk 2 above)
Comment (cptbtptpbcptdtptp) [reply docs/zh/xr/quickStart/develop.md:110]: Done.

File: docs/zh/xr/quickStart/debug.md (hunk 1 above)
Comment (cptbtptpbcptdtptp) [reply docs/zh/xr/quickStart/debug.md:57]: Done.

File: docs/zh/xr/system/features.md (hunk 3 above)
Comment (cptbtptpbcptdtptp) [reply docs/zh/xr/system/features.md:100]: 可以直接扫码体验，我补充一下追踪的图片

File: docs/zh/xr/system/features.md [hunk 4]
```diff
@@ -89,13 +85,19 @@ xrManager.addFeature(XRPlaneTracking, XRPlaneMode.EveryThing);
 > 需要注意的是，图片追踪在添加功能时就需要指定追踪的图片，并且在 WebXR 中，同张图片只会被追踪一次。
 
 ```typescript
-// 在初始化时指定平面追踪的类型为所有
-xrManager.addFeature(XRImageTracking, [refImage]);
+const image = new Image();
+image.onload = () => {
+  // 创建追踪图片
+  const refImage = new XRReferenceImage("test", image, 0.08);
```
[the remark this answers is not in the document]
Comment (cptbtptpbcptdtptp) [reply docs/zh/xr/system/features.md:99]: Done.

**Pull request merged**